package.json

```diff
@@ -47,6 +48,8 @@
     "uglifyjs-webpack-plugin": "2.1.1",
     "webpack": "4.28.3",
     "webpack-cli": "3.2.0",
-    "webpack-merge": "4.2.1"
+    "archiver": "^3.1.1",
```

Comment:
deps should be alphabetized (this will happen automatically if you install from the command line via `npm i -D <package_name>`)

Reply:
oops. I accidentally installed them as regular deps and moved them manually wich I forgot to alphabetize.